dipy/align/tests/test_transforms.py

```diff
@@ -0,0 +1,230 @@
+from dipy.align.transforms import regtransforms
+import numpy as np
+from numpy.testing import (assert_array_equal,
+                           assert_array_almost_equal,
+                           assert_almost_equal,
+                           assert_equal,
+                           assert_raises)
+
+def test_number_of_parameters():
+    expected_params = {('TRANSLATION', 2) : 2,
+                       ('TRANSLATION', 3) : 3,
+                       ('ROTATION', 2) : 1,
+                       ('ROTATION', 3) : 3,
+                       ('RIGID', 2) : 3,
+                       ('RIGID', 3) : 6,
+                       ('SCALING', 2) : 1,
+                       ('SCALING', 3) : 1,
+                       ('AFFINE', 2) : 6,
+                       ('AFFINE', 3) : 12}
+
+    for ttype, transform in regtransforms.items():
+        assert_equal(transform.get_number_of_parameters(), expected_params[ttype])
+
+
+def test_param_to_matrix_2d():
```

Comment:
Consider having separate random number generator here in case you want to seed it later?

```
rng = np.random.RandomState()
```

Then use size for multiple values, eg:

```
angle, dx, dy = rng.normal(size=(3,))
```